bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/KonnectedBindingConstants.java

```diff
@@ -36,7 +36,7 @@ public class KonnectedBindingConstants {
     public static final ThingTypeUID THING_TYPE_PROMODULE = new ThingTypeUID(BINDING_ID, PRO_MODULE);
 
     // Thing config properties
-    public static final String HOST = "ipAddress";
+    public static final String BASE_URL = "base_url";
```

Comment:
Since backwards compatibility is already broken, could this be **camelCase** like ipAddress was and macAddress still is?

Reply:
I have the same remark until I see that others are not camelCase !

Reply:
This could be the opportunity for renaming them too. :)

Reply:
I renamed all config parameters to camel case for consistency.